tts-deploy-on-embedded.ipynb

```diff
@@ -0,0 +1,265 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "caa47ff5",
+   "metadata": {},
+   "source": [
+    "# TTS Deploy on embedded\n",
+    "\n",
+    "In this tutorial we will explain the process of generating TTS RMIR from synthesizer.riva and vocoder.riva files for embedded machines. RMIR (Riva Model Intermediate Representation) is an intermediate file that has all the necessary artifacts (models, files, configurations, and user settings) required to deploy a Riva service.  \n",
+    "\n",
+    "This tutorial assumes that the .riva files are present for both synthesizer and vocoder. [`nemo2riva`](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/model-overview.html#export-models-with-nemo2riva) can be used to generate .riva files from nemo checkpoints.  \n",
```

Comment:
This notebook doesn't assume that the .riva files are present, it downloads them from NGC. It might be more helpful to point that we use this NGC as a demo here instead of further down.